### ISSUE TITLE
Rotate logstash's containerid-json.log file

### ISSUE THREAD

**Issue by etique57** [deviantony/docker-elk#285]
Hello,
I hope this is the proper place to log this issue.

### Problem description

In my logstash container, there's a file named cotainerid-json.log  that grows continuously up to a point it fills up my (small) system disk.

I couldn't find it from within the docker.

#### Stack configuration

Problem was on 6.2.3, just switched to 6.2.4

#### Docker setup

```
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:17:38 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:15:45 2018
  OS/Arch:      linux/amd64
  Experimental: false
```

```
docker-compose version 1.21.2, build a133471
docker-py version: 3.3.0
CPython version: 3.6.5
OpenSSL version: OpenSSL 1.0.1t  3 May 2016
```

#### Docker logs

```
huge output, I don't dare to paste it here!
```

**Comment by antoineco** [comment]
@etique57 you're talking about the way Docker redirects the log output of your containers, which is `json-file` by default: https://docs.docker.com/config/containers/logging/configure/#use-environment-variables-or-labels-with-logging-drivers

These pages explain how to limit the size of container logs, globally or per container:

- https://docs.docker.com/config/containers/logging/json-file/
- https://docs.docker.com/compose/compose-file/#logging

Long story short:
```yaml
# docker-compose.yml

# ...

  logstash:
    logging:
      driver: "json-file"
      options:
        max-size: "5m"
        max-file: "3"
```
